SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitLauncherCallback.java

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.NonNull;
+
+public interface SEPADirectDebitLauncherCallback {
```

Comment:
This will need docs (I think I missed them in PayPal too), but since the params/description will likely change based on the decision in #813 I think it's fine to leave blank for now

Reply:
Good catch! I gave it a shot and added a docstring in 16d1fcd0e3995d2d6804c4c4b525e85eda3e5973